steam_wrapper/steam_wrapper.py

```diff
@@ -211,7 +262,8 @@ def configure_shared_library_guard():
 
 def main(steam_binary=STEAM_PATH):
     os.chdir(os.environ["HOME"]) # Ensure sane cwd
-    print ("https://github.com/flathub/com.valvesoftware.Steam/wiki")
+    logging.basicConfig(level=logging.DEBUG)
```

Comment:
Why DEBUG by default?

Reply:
We don't (and can't) have CLI options to increase verbosity, to I thought we should log everything by default.

Reply:
Well, we could have an environment variable.

Reply:
I suppose this can be tweaked later.